calabash/Classes/FranklyServer/Routes/LPBackdoorRoute.m

```diff
@@ -29,40 +34,41 @@ - (NSDictionary *) JSONResponseForMethod:(NSString *) method URI:(NSString *) pa
     selectorName = [selectorName stringByAppendingString:@":"];
   }
 
+  id argument = [data objectForKey:@"arg"];
+  if (!argument) {
+    LPLogError(@"Expected data dictionary to contain an 'arg' key");
```

Comment:
Meta-language manipulations is one of my favorite things ever. I'm _sure_ there's a way to do it in a way that's non-breaking. 

As soon as we hit the `NSSelectorFromString()` call, the number and type of arguments shouldn't matter (at least insofar as the ObjcRuntime is concerned). Looks like it's more a question of how `LPInvoker` can handle `nil`s.